### PR TITLE
(update) login process, request process

### PR DESCRIPTION
**Pull request by mconner89**

### CODE REVIEW

File: server/db/routes/requestRoutes.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable linebreak-style */
```
Comment (jstrzesz) [inline server/db/routes/requestRoutes.js]: remove es-lint disable

File: server/db/routes/userRoutes.js [hunk 2]
```diff
@@ -43,12 +43,42 @@ userRouter
   })
   .get('/email/:address', (req, res) => {
     const { address } = req.params;
+    console.log('here?');
```
Comment (jstrzesz) [inline server/db/routes/userRoutes.js]: remove

File: server/db/routes/userRoutes.js [hunk 3]
```diff
@@ -43,12 +43,42 @@ userRouter
...
     User.findOne({
       where: {
         email: address,
       },
     })
-      .then((user) => res.send(user))
+      .then((response) => {
+        console.log(response, 'wheres my fucking response');
```
Comment (jstrzesz) [inline server/db/routes/userRoutes.js]: remove. Additionally, keep language in commits professional.

File: server/routes/dashboard.js [hunk 4]
```diff
@@ -3,17 +3,15 @@ const authorize = require("../utils/authorize");
 const { db, User } = require("../db/index");
 
 router.post("/", authorize, async (req, res) => {
+  const { email } = req.headers;
+  // console.log(req, 'body in post route from dashboard.js');
```
Comment (jstrzesz) [inline server/routes/dashboard.js]: remove

File: server/routes/jwtAuth.js [hunk 5]
```diff
@@ -1,16 +1,18 @@
...
-router.post("/register", async (req, res) => {
+// Signup/Register
+router.post('/register', async (req, res) => {
+  // 1. Destructure the req.body (name, email, password)
+  // Change back to camel case
   const { firstName, lastName, email, password } = req.body;
   try {
     console.log(req.body);
-    console.log("Entered variables", firstName, lastName, email, password);
+    console.log('Entered variables', firstName, lastName, email, password);
```
Comment (jstrzesz) [inline server/routes/jwtAuth.js]: remove console.logs

File: src/components/landing/auth/Login.tsx [hunk 6]
```diff
@@ -42,12 +63,17 @@ const Login: React.FC<AuthProps> = ({ handleLogin: [isAuthenticated, setAuth] })
           // Accept: 'application/json,text/plain, */*',
         },
         body: JSON.stringify(body),
-      });
+      })
+        .then((data) => {
+          console.log(data, 'data from login.tsx line 40');
```
Comment (jstrzesz) [inline src/components/landing/auth/Login.tsx]: remove

File: src/components/landing/auth/Login.tsx [hunk 7]
```diff
@@ -42,12 +63,17 @@ const Login: React.FC<AuthProps> = ({ handleLogin: [isAuthenticated, setAuth] })
...
+          return data;
+        });
 
       const parseRes = await response.json();
       console.log(parseRes.jwtToken);
```
Comment (jstrzesz) [inline src/components/landing/auth/Login.tsx]: remove

File: src/components/landing/auth/Login.tsx [hunk 8]
```diff
@@ -42,12 +63,17 @@ const Login: React.FC<AuthProps> = ({ handleLogin: [isAuthenticated, setAuth] })
...
       if (parseRes.jwtToken) {
         localStorage.setItem('token', parseRes.jwtToken);
+        getUserProfile();
         loginUser();
         console.log('Logged in? ', isAuthenticated);
```
Comment (jstrzesz) [inline src/components/landing/auth/Login.tsx]: remove

File: src/components/landing/auth/Signup.tsx [hunk 9]
```diff
@@ -91,6 +107,7 @@ const SignUp: React.FC<AuthProps> = ({ handleLogin: [isAuthenticated, setAuth] }
       if (parseRes.jwtToken) {
         console.log(parseRes.jwtToken);
```
Comment (jstrzesz) [inline src/components/landing/auth/Signup.tsx]: remove

**Review by jstrzesz — CHANGES_REQUESTED** [review]
all console.logs must be removed. Also, remove vulgar language from code and commits.

**Review by jessrichmond — APPROVED** [review]

**Review by gregthompson27 — APPROVED** [review]

**Pull request merged**